portal/templates/portal/play_aimmo.html

```diff
@@ -36,10 +36,10 @@ <h1 class="banner__text--primary">Level 1 prototype</h1>
 
 <div class="background">
     <section>
-        <h1 class="col-sm-7 col-center">Code for Life will be launching a new game for secondary schools in 2018!</h1>
+        <h1 class="col-sm-7 col-center">Code for Life will be launching a new game for secondary schools soon!</h1>
         <h4 class="col-sm-6 col-center">Following on from Rapid Router, this exciting game will build on students’ existing coding skills.</h4>
-        <h4 class="col-sm-6 col-center">The game will be a time travelling adventure, which will take them to the next level on their coding journey.
-            We’ll have a playable beta of Level 1 by early 2018, but first we’d like to get your thoughts on the gameplay, learning concept and general design.</h4>
+        <h4 class="col-sm-6 col-center">The game will be a Massively Multiplayer Online (MMO) time travelling adventure, which will take them to the next level on their coding journey: Artificial Intelligence and Python.
+            We’ll have a playable beta soon, but first we’d like to get your thoughts on the gameplay, learning concept and general design.</h4>
```

Comment:
is there a contact us button / email on this page we could use? (i.e. a clear call to action)